app/views/amusement_parks/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<p>Name: <%= @amusement_park.name %></p>
+<p>Cost of Admission: <%= @amusement_park.admission_cost %></p>
+
+<h4>Mechanics:</h4>
+<ul>
+  <% @mechanics.each do |mechanic| %>
```

Comment:
It is here that you would call the method that you create in your `amusement_park` model (see my comments in that file). So it would look something like:
```ruby
@amusement_park.unique_mechanics.each do |mechanic|
```